pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -901,7 +903,22 @@ func validateBootOrder(field *k8sfield.Path, spec *v1.VirtualMachineInstanceSpec
 			}
 			bootOrderMap[order] = true
 		}
+
+		diskNames[disk.Name] = nil
 	}
+
+	// Validate that volumes match disks correctly
+	for idx, volume := range spec.Volumes {
```

Comment:
OK! this ensures there's always a disk if a volume exists.

Is the reflexive case required? Do we need a volume if a disk exists?

Reply:
The reflexive case is already being checked a few lines above (line 877) and there also a test covering it